.gitmodules

```diff
@@ -17,3 +14,7 @@
 	path = AI/Skirmish/BARb
 	url = https://github.com/rlcevg/CircuitAI.git
 	branch = barbarian
+[submodule "AI/Skirmish/CircuitAI"]
+	path = AI/Skirmish/CircuitAI
+	url = https://github.com/rlcevg/CircuitAI.git
```

Comment:
please revert this change. we had it several times that required submodules disappeared because of renames or deletion. thats why all submodules are hosted at https://github.com/spring/*